visualization/include/pcl/visualization/pcl_visualizer.h

```diff
@@ -1200,7 +1200,19 @@ namespace pcl
         setShapeRenderingProperties (int property, double value,
                                      const std::string &id, int viewport = 0);
 
-        /** \brief Set the rendering properties of a shape (3x values - e.g., RGB)
+        /** \brief Set the rendering properties of a shape (2x values - e.g., LUT minmax values)
+          * \param[in] property the property type
+          * \param[in] val1 the first value to be set
+          * \param[in] val2 the second value to be set
+          * \param[in] id the shape object id
+          * \param[in] viewport the view port where the shape's properties should be modified (default: all)
```

Comment:
Viewport is in pretty much all the functions but sometimes it's not implemented, which is is bad because it lures the user.

Changing the `int` `property` should be done in an other PR in my opinion because it must be done everywhere in PCLVisualizer code (including `interactor_style.cpp`)